pkg/gui/presentation/files.go

```diff
@@ -158,8 +159,10 @@ func getFileLine(hasUnstagedChanges bool, hasStagedChanges bool, name string, di
 	isLinkedWorktree := file != nil && file.IsWorktree
 	isDirectory := file == nil
 
+    var testColor = color.C256(74, false)
+    // var testColor= style.FgGreen
 	if icons.IsIconEnabled() {
-		output += restColor.Sprintf("%s ", icons.IconForFile(name, isSubmodule, isLinkedWorktree, isDirectory))
+		output += testColor.Sprintf("%s ", icons.IconForFile(name, isSubmodule, isLinkedWorktree, isDirectory))
```

Comment:
Try
```go
output += testColor.Sprint(icons.IconForFile(name, isSubmodule, isLinkedWorktree, isDirectory)) + " "
```

Reply:
That is better but doesn't trace the logo, I wonder if you get icons that are already colored.
<img width="428" alt="Screenshot 2023-10-04 at 4 47 08 PM" src="https://github.com/jesseduffield/lazygit/assets/42164334/ef5f6e36-d8c0-4a3e-8862-7de2f6a9f558">

Reply:
https://github.com/nvim-tree/nvim-web-devicons/blob/master/lua/nvim-web-devicons.lua going off this for the colors, mit license, but it is in lua

Reply:
Looks fine to me: 
<img width="491" alt="image" src="https://github.com/jesseduffield/lazygit/assets/8456633/61e70ec6-45c6-4b9d-b3a4-d9e1bbb36e91">

Reply:
ah thats what I was asking should it be a fg color or a bg color. Most applications do a bg, but fg works perfectly if that is consensus

Reply:
This was example from the issue creator so I assume that what we wanted to go for

<img width="349" alt="Screenshot 2023-10-04 at 4 59 04 PM" src="https://github.com/jesseduffield/lazygit/assets/42164334/c3f29c53-2559-455f-8935-3cfa52b3b030">

Reply:
hmm now I'm confused. Are you sure other apps use a background colour? Maybe we just need to use a different icon that's filled out rather than just an outline

Reply:
atleast with nvim tree they do specify a color so that is what I assuming is setting the color of the icon. I think looking for color icons may be a way to go though.

Reply:
if in pkg/gui/presentation/icons/file_icons.go we use:
```go
".go":             "\ue627",     // 
```
And continue using the foreground colour, we get:
<img width="601" alt="image" src="https://github.com/jesseduffield/lazygit/assets/8456633/2aaaa3bd-085d-4ac5-b1c2-bae372ac6923">

So looks like we just need to update some icons

Reply:
oh awesome, I will look into that then thank you!

Reply:
was the string just from nerd fonts?